src/backend/functions/risk-edit.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import middy from '@middy/core';
+import jsonBodyParser from '@middy/http-json-body-parser';
+import httpErrorHandler from '@middy/http-error-handler';
+import validator from '@middy/validator';
+import { Handler } from 'aws-lambda';
+import { PrismaClient, Role } from '@prisma/client';
+import {
+  buildNoAuthResponse,
+  buildNotFoundResponse,
+  buildSuccessResponse,
+  eventSchema,
+  riskEditInputSchemaBody
+} from 'utils';
+import { FromSchema } from 'json-schema-to-ts';
+
+const prisma = new PrismaClient();
+
+export const editRisk: Handler<FromSchema<typeof inputSchema>> = async ({ body }, _context) => {
+  const { userId, id, detail, resolved } = body;
+
+  // verify user is allowed to edit work packages
+  const user = await prisma.user.findUnique({ where: { userId } });
+  if (!user) return buildNotFoundResponse('user', `#${userId}`);
+  if (user.role === Role.GUEST) return buildNoAuthResponse();
+
+  // get the original risk and check if it exists
+  const originalRisk = await prisma.risk.findUnique({
+    where: { id }
+  });
+  if (originalRisk === null) {
+    return buildNotFoundResponse('Risk', `#${id}`);
+  }
+
+  // update the risk with the input fields
+  const updatedRisk = await prisma.risk.update({
+    where: {
```

Comment:
Prettier formatting

Reply:
Try putting each clause (where, data) on one line each.

```
 where: { id },
 data: { id, detail }
```

I believe both are valid for prettier, but one is fewer lines of code and thus more compact.